src/vsr/replica_test.zig

```diff
@@ -1514,25 +1514,22 @@ test "Cluster: client: empty command=request operation=register body" {
         .request = 0,
         .command = .request,
         .operation = .register,
-        .release = releases[0].release,
+        .release = vsr.Release.from(.{ .major = 0, .minor = 15, .patch = 3 }),
```

Comment:
```suggestion
        .release = vsr.Release.from(.{ .major = 0, .minor = 0, .patch = 9 }),
```

With this change we will fail with `client_release_too_low`,  which reflects how we actually expect it to work.